src/main/java/org/jboss/pull/player/Util.java

```diff
@@ -88,7 +90,20 @@ static boolean optionalBoolean(final String name, final boolean defaultValue) {
     private static class PropertiesHolder {
         static final Properties PROPERTIES = new Properties();
         static {
-            try (Reader reader = Files.newBufferedReader(BASE_DIR.toPath().resolve("player.properties"), StandardCharsets.UTF_8)) {
+            File[] children = BASE_DIR.listFiles((dri, name) -> name.endsWith("player.properties"));
+            if (children != null) {
+                Arrays.stream(children).sorted((o1, o2) -> {
+                    // Plain player.properties goes first; other files then override
+                    int res = o1.compareTo(o2);
+                    return res > -1 || "player.properties".equals(o2.getName()) ? res : 1;
+                }).forEach(PropertiesHolder::loadPlayerProperties);
+            } else {
+                throw new IllegalStateException(String.format("%s has no player.properties files", BASE_DIR));
+            }
+        }
+
+        private static void loadPlayerProperties(File properties) {
+            try (Reader reader = Files.newBufferedReader(properties.toPath(), StandardCharsets.UTF_8)) {
```

Comment:
A general comment at the end: It would seem to me to be easier to name them player.properties, then perhaps look for player.properties-1, -2, -3 etc  vs just assuming ordering, but perhaps I've misunderstood.